### PR TITLE
Merge Introduction Page Initial Structure & Content

### PR DESCRIPTION
**Pull request by samuelsogbesan**
**- Add the initial HTML Structure for the introduction page. The HTML structure is intended to reflect the structure shown in the draft below:**
image

- **Add the accompanying CSS (absent of colour) to the structure. Here, CSS Flexbox is used to format the content on the page.**

Awaiting the merge of PR #5, colour and font styling will be implemented in a future PR. 

Current Look: 
image

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,10 +7,34 @@
...
+          <div>
+            <h2>Samuel Sogbesan</h2>
+            <div class="introduction-image-wrapper">
+              <p>
+                I’m a <b>STEP Intern</b> at Google at the moment...<br>
+                I’m also a <b>student</b> at the <b>University of Bath</b><br>
+                I study <b>Computer Science</b> and I’m in my 2nd Year!
+                I’m into <b>Web Development</b><br>
+                I listen to <b>Hip-Hop, Rap, R&B</b><br>
+                I play <b>video game</b> (FPS!)
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: nit: not sure if you meant "video game**s**"?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -7,10 +7,34 @@
...
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+    <div id="fullpage">
+      <div class="section"></div>
+      <div class="section" id="introduction">
+        <div class="introduction-wrapper">
+          <div>
+            <h3>Hey I'm</h3>
+            <div class="introduction-image-wrapper">
+              <img src="https://media-exp1.licdn.com/dms/image/C5603AQHvj5cLWuV6Tw/profile-displayphoto-shrink_200_200/0?e=1598486400&v=beta&t=ozRZ9uW0azRIoXdce5BafIl0VyLcY0XLbY26LwYl_GE">
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: [please provide multimedia fallback](https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback)

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
+body{
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: reformat the css according to https://google.github.io/styleguide/htmlcssguide.html#CSS_Style_Rules

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/index.html]: Aha, I definitely did! Pushed a correction with commit 34f9279

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/index.html]: Thanks, corrected in commit a47d775

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/index.html]: Definitely did! Corrected in commit 34f9279

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Thanks

- Shortened 'introduction' to 'intro' for ID and classes in commits e3c334d and 943758c ([Rule 4.1.2](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming))

- Added space between '{' and selector for all selectors in commit 6563d34 ([Rule 4.2.5](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation))

Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: - Added a new line for selector decleration in commit 6563d34 ([Rule 4.2.6](https://google.github.io/styleguide/include/link.png))

Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: - Added Changed tab spacing to +2 and separated rules with line break in commit 
ebef3af ([Rule 4.2.7](https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation))

Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: - Made selector name more generalized and purposeful in commit c33f39e ([Rule 4.1.2](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming))

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
+body {
+  margin:0;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: add a space after `:`, here and below

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  width: 70%;
+  height: 50vh;
+  text-align:center;
+}
+
+.intro-wrapper > * {
+  display: flex;
+  flex-direction: column;
+  flex: 0.5;
+  align-items: center;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: [alphabetize declarations](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order)

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+.intro-wrapper > * > * {
+  display:flex;
+  align-items:center;
+  justify-content: center;
+}
+
+.intro-wrapper h3 {
+  font-family: "Roboto Light";
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: [use the `font` shorthand property instead](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties)

File: portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+.intro-wrapper h2 {
+  margin:0;
+  font-size:3rem;
+  line-height: 2.5rem;
+}
+
+.intro-wrapper p {
+  background-color:white;
+  text-align: center;
+}
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css:80]: add a blank line after this, for consistency with the rest of the file

Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css:80]: Corrected with commit 5bf34d3

File: portfolio/src/main/webapp/style.css (hunk 6 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Corrected with commit 457f8a5

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Amended when merged from master d4cac10

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Alphabetised in commit c4c4699

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -25,6 +25,64 @@ body {
...
+#intro,
+.intro-wrapper {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.intro-wrapper {
+  height: 50vh;
+  text-align:center;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: add a space after `:`, here and elsewhere where it is missing

**Review by krasimirgg — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/style.css (hunk 8 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Rectified here and elsewhere in commit ab13304

**Review by krasimirgg — APPROVED** [review]
Awesome!

**Pull request merged**